db/item_db.conf

```diff
@@ -150872,6 +150872,27 @@ item_db: (
 
 //== New Etc Items =========================================
 {
+	Id: 22508
+	AegisName: "Para_Team_Mark_"
+	Name: "Eden Group Mark"
+	Type: 11
+	Buy: 0
+	Delay: 1200000
+	Trade: {
+		nodrop: true
+		notrade: true
+		noselltonpc: true
+		nocart: true
+		nostorage: true
+		nogstorage: true
+		nomail: true
+		noauction: true
+	}
+	Script: <"
+		unitskilluseid getcharid(3),"AL_TELEPORT",3;
```

Comment:
I have not access to kRO server, but I check in navigation data and there is this info:

	{ "moc_para01", 10349, 101, 941, "마녀 민트", "2016VE", 184, 36 },
	{ "moc_para01", 10350, 101, 876, "와플", "2016VE", 186, 36 },
	{ "moc_para01", 10351, 101, 88, "밀가루 상인", "2016VE", 187, 30 },
	{ "moc_para01", 17170, 101, 704, "카탈로그 마도사", "catal01", 22, 16 },
	{ "moc_para01", 17257, 101, 554, "냥타임", "para", 29, 171 },
	{ "moc_para01", 22079, 101, 564, "초보자용 자판기", "낙원단용", 17, 37 },
	{ "moc_para01", 22240, 101, 899, "정리의 달인", "", 165, 51 },
	{ "moc_para01", 22568, 101, 469, "낙원단교관 보야", "para01", 25, 35 },
	{ "moc_para01", 22575, 101, 967, "관리자 미하엘", "para09", 112, 96 },
	{ "moc_para01", 22576, 101, 820, "요리사", "", 179, 44 },
	{ "moc_para01", 22577, 101, 468, "낙원단 교관 우르", "2nd01", 23, 35 },
	{ "moc_para01", 22586, 101, 813, "대장장이 토른", "2nd10", 111, 83 },
	{ "moc_para01", 22587, 101, 851, "무기전문가 비케이", "2nd11", 112, 79 },
	**{ "moc_para01", 22588, 101, 952, "사무담당 라임 이베노르", "eden", 27, 35 },**
	{ "moc_para01", 22700, 101, 900, "늙은 모험가", "eden", 16, 22 },
	{ "moc_para01", 22701, 101, 886, "낙원단 요리사", "eden", 182, 48 },
	{ "moc_para01", 22702, 101, 883, "수습 낙원단원", "eden", 172, 28 },
	{ "moc_para01", 22703, 101, 814, "사무보조 니데", "eden", 20, 35 },
	{ "moc_para01", 22704, 101, 555, "관리자 하모니", "para01", 112, 90 },
	{ "moc_para01", 22705, 101, 899, "중개의뢰 보상담당원", "qc", 31, 31 },
	{ "moc_para01", 22706, 101, 857, "1~39 의뢰게시판", "낙원단", 34, 38 },
	{ "moc_para01", 22707, 101, 857, "40~69 의뢰게시판", "낙원단", 36, 38 },
	{ "moc_para01", 22708, 101, 857, "70~99 의뢰게시판", "낙원단", 38, 38 },
	{ "moc_para01", 22709, 101, 995, "낙원단 물류담당자", "낙원단", 29, 35 },
	{ "moc_para01", 22710, 101, 875, "레노", "remP", 50, 23 },
	{ "moc_para01", 22711, 101, 635, "입실론 r. 카오스", "remP", 45, 183 },
	{ "moc_para01", 22712, 101, 641, "류리아 사르휘나", "remP", 43, 179 },
	{ "moc_para01", 22713, 101, 10165, "로제 카르보", "remP", 48, 175 },
	{ "moc_para01", 22714, 101, 88, "성실한 소길", "remP", 49, 180 },
	{ "moc_para01", 22834, 101, 629, "겔카", "1", 37, 95 },
	{ "moc_para01", 22295, 101, 630, "로테르트", "12", 41, 93 },
	{ "moc_para01", 22296, 101, 643, "밍민", "1", 19, 95 },
	{ "moc_para01", 22297, 101, 648, "멜로디-잭", "1", 44, 81 },
	{ "moc_para01", 22298, 101, 646, "아이구", "1", 19, 78 },
	{ "moc_para01", 22305, 101, 647, "라기", "1", 41, 79 },

The one in black 사무담당 라임 이베노르 I guess is Secretary Lime Evenor, but I cannot confirm it because we have not live access to kRO.

The npc changing is this, Secretary Lime Evenor in moc_para01,27,35, since she is the one who gives the new Eden Group Mark and changes the old one for this new one, according to a live test in iRO, where we have some old versions of Eden Group Marks.